ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/DC1_TENANTS_NETWORKS/Tenant_A.yml

```diff
@@ -313,3 +315,19 @@ tenant_a:
         name: overlapping_name
       - id: 165
         name: overlapping_name
+
+ipv4_acls:
+  - name: TEST-IPV4-ACL-WITH-IP-FIELDS-IN
+    entries:
+      - sequence: 15
+        action: deny
+        protocol: ip
+        source: any
+        destination: interface_ip
```

Comment:
While setting source or destination as peer_ip then getting below error. Need to check more:
```
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ansible_collections.arista.avd.plugins.plugin_utils.errors.errors.AristaAvdError: Unable to perform substitution of the value 'peer_ip' defined under 'ipv4_acls[name=TEST1-IPV4-ACL-WITH-IP-FIELDS-IN].entries[0].source', since no substitution value was found for interface 'Tenant_A_OP_Zone_1'. Make sure to set the appropriate fields on the interface.
fatal: [DC1-SVC3B -> localhost]: FAILED! => {"changed": false, "msg": "Unable to perform substitution of the value 'peer_ip' defined under 'ipv4_acls[name=TEST1-IPV4-ACL-WITH-IP-FIELDS-IN].entries[0].source', since no substitution value was found for interface 'Tenant_A_OP_Zone_1'. Make sure to set the appropriate fields on the interface."}
```

Reply:
peer_ip is not applicable for SVI.

Reply:
The error message seems misleading. It should mention the interface `Vlan123`.